app/models/html_document.rb

```diff
@@ -41,10 +41,21 @@ def parse_content
     # it to html, then running it through Loofah again, it's because Loofah's 'to_text' method is
     # much smarter about whitespace than Nokogiri, but you can't run `to_text` on the Nokogiri
     # elements extracted by "html.at('main')". Hence the jumping through hoops.
-    plain_text = Loofah.fragment(main_html).scrub!(:whitewash).to_text(encode_special_chars: false)
+    plain_text = Loofah.fragment(main_html).
+      scrub!(tag_scrubber).
+      scrub!(:whitewash).
+      to_text(encode_special_chars: false)
     plain_text.gsub(/[ \t]+/,' ' ).gsub(/[\n\r]+/, "\n").chomp.lstrip
   end
 
+  def tag_scrubber
+    # convert custom tags to plain 'ol divs to ensure they are not
+    # stripped out during HTML sanitization
+    Loofah::Scrubber.new do |node|
+      node.name = 'div' if /-/ === node.name
+    end
```

Comment:
Is the === necessary here? https://github.com/rubocop-hq/ruby-style-guide#no-case-equality